src/components/Button/Button.module.scss

```diff
@@ -25,7 +24,18 @@ $disabled-on-primary-color-opacity: 0.5;
   user-select: none;
   /* Will be updated dynamically after render by js */
   --element-width: 32;
-  --element-height: 32;
+  min-width: var(--element-width);
+  width: fit-content;
```

Comment:
is there really need in min-width and initial 32 width? it's also without `px` so it probably has no affect and it is immediately overriden.